src/main/java/com/scalar/database/transaction/consensuscommit/RecoveryHandler.java

```diff
@@ -17,7 +17,11 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-/** */
+/**
+ * A recovery handler for transactions on {@link DistributedStorage}. Used to rollback records on
+ * {@link DistributedStorage} to a given {@link Snapshot} or to recover a given {@link
```

Comment:
It will recover the record on the storage, not a given result.

Reply:
Thank you! Updated it somewhat.